packages/desktop-client/src/style/tags.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useCallback } from 'react';
```

Comment:
We should create a feature folder for notes in `desktop-client/src/notes` and put these components there